pkg/common/config/types.go

```diff
@@ -57,6 +57,13 @@ type Config struct {
 		Zone   string `gcfg:"zone"`
 		Region string `gcfg:"region"`
 	}
+	// Set of features with state flags
+	FeatureStates FeatureStateSwitches
```

Comment:
Does this match the property we added for supersior and guest cluster?

Reply:
Yes, it is added for the supervisor and guest cluster too. Once @RaunakShah brings in changes here on Github, we can see it.